### PR TITLE
dev-util/rgbds: enable tests

### PR DESCRIPTION
**Pull request by orbea**

### CODE REVIEW

File: dev-util/rgbds/rgbds-9999.ebuild [hunk 1]
```diff
@@ -36,3 +36,12 @@ src_install() {
 	emake DESTDIR="${D}" PREFIX="${EPREFIX}"/usr Q= STRIP= install
 	dodoc README.md
 }
+
+src_test() {
```
Comment (thesamesam) [inline dev-util/rgbds/rgbds-9999.ebuild:40]: On merge, I'll update this to reflect phase execution order (i.e. put src_test above src_install).

**Comment by thesamesam** [comment]
Can you check the app-alternatives/yacc[bison] dep? It's a bit odd -- if it really needs bison, it should call bison, not rely on the yacc symlink _being_ bison (or allow overriding it at least).

**Comment by orbea** [comment]
> Can you check the app-alternatives/yacc[bison] dep? It's a bit odd -- if it really needs bison, it should call bison, not rely on the yacc symlink being bison (or allow overriding it at least).

Yea, I am not sure what I was thinking when I did that, it needs bison, but doesn't use yacc... I'll make a PR to fix this shortly...

**Comment by orbea** [comment]
> Yea, I am not sure what I was thinking when I did that, it needs bison, but doesn't use yacc... I'll make a PR to fix this shortly...

And done. https://github.com/gentoo/gentoo/pull/35856

**Pull request closed without merging**